src/main/java/mediator/Mediator.java

```diff
@@ -66,7 +66,6 @@ public static Mediator getInstance() {
 	 */
 	private void printRegisteredRadioUnits() {
 		List<ManagedRadioUnit> radioUnits = radioUnitRegistry.getAllRadios();
-		radioUnits.forEach(System.out::println);
 		if (radioUnits.size() == 0) {
 			System.out.println("[ERROR] No RUs have been registered with the system.");

```

Comment:
Small note: I see that in this printRegisteredRadioUnits method, we have an error message if there aren't actually any radios. However, I do not see any such message in the other List methods (e.g. if we run option 17 [List RUs by standard (RAT type)], it will just print nothing if there aren't any radios of that RAT type).

Reply:
This is also something fixed in my other pull request.